### ISSUE TITLE
Uncaught Error: Cannot call function 'getPosition' on missing remote object 52Error: Cannot call...

### ISSUE THREAD

**Issue by borishere** [atom/atom#16840]
[Enter steps to reproduce:]

1. Build .scss file
2. It happened only once, can't reproduce it 100% times

**Atom**: 1.24.0 x64
**Electron**: 1.6.16
**OS**: Microsoft Windows 10 Enterprise N 2015 LTSB
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot call function 'getPosition' on missing remote object 52
Error: Cannot call function 'getPosition' on missing remote object 52
    at throwRPCError (C:\Users\user\AppData\Local\atom\app-1.24.0\resources\electron.asar\browser\rpc-server.js:143:17)
    at EventEmitter.<anonymous> (C:\Users\user\AppData\Local\atom\app-1.24.0\resources\electron.asar\browser\rpc-server.js:354:7)
    at emitMany (events.js:127:13)
    at EventEmitter.emit (events.js:201:7)
    at WebContents.<anonymous> (C:\Users\user\AppData\Local\atom\app-1.24.0\resources\electron.asar\browser\api\web-contents.js:256:37)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)

```
At C:\Users\user\AppData\Local\atom\app-1.24.0\resources\electron.asar\renderer\api\remote.js:234

Error: Cannot call function 'getPosition' on missing remote object 52
Error: Cannot call function 'getPosition' on missing remote object 52
    at throwRPCError (~/AppData/Local/atom/app-1.24.0/resources/electron.asar/browser/rpc-server.js:143:17)
    at EventEmitter.<anonymous> (~/AppData/Local/atom/app-1.24.0/resources/electron.asar/browser/rpc-server.js:354:7)
    at emitMany (events.js:127:13)
    at EventEmitter.emit (events.js:201:7)
    at WebContents.<anonymous> (~/AppData/Local/atom/app-1.24.0/resources/electron.asar/browser/api/web-contents.js:256:37)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)
    at metaToValue (~/AppData/Local/atom/app-1.24.0/resources/electron.asar/renderer/api/remote.js:234:13)
    at Object.remoteMemberFunction (~/AppData/Local/atom/app-1.24.0/resources/electron.asar/renderer/api/remote.js:118:18)
    at AtomEnvironment.getWindowDimensions (~/AppData/Local/atom/app-1.24.0/resources/app/src/atom-environment.js:690:40)
    at AtomEnvironment.storeWindowDimensions (~/AppData/Local/atom/app-1.24.0/resources/app/src/atom-environment.js:727:40)
    at WindowEventHandler.handleWindowBlur (~/AppData/Local/atom/app-1.24.0/resources/app/src/window-event-handler.js:189:32)
```

### Commands

```
     -1:11.8.0 snippets:next-tab-stop (input.hidden-input)
     -1:11.8.0 snippets:expand (input.hidden-input)
     -1:11.8.0 editor:indent (input.hidden-input)
     -1:08.5.0 core:backspace (input.hidden-input)
     -0:54.8.0 autocomplete-plus:confirm (input.hidden-input)
     -0:53.3.0 core:move-right (input.hidden-input)
     -0:51.8.0 editor:newline (input.hidden-input)
  2x -0:50.1.0 core:move-down (input.hidden-input)
     -0:48.5.0 editor:move-to-end-of-screen-line (input.hidden-input)
     -0:48.1.0 intentions:highlight (input.hidden-input)
     -0:47.7.0 bracket-matcher:select-inside-brackets (input.hidden-input)
  2x -0:45.5.0 editor:move-line-up (input.hidden-input)
     -0:44.3.0 core:move-down (input.hidden-input)
     -0:43.8.0 intentions:highlight (input.hidden-input)
     -0:43.7.0 core:cut (input.hidden-input)
     -0:42.7.0 build:trigger (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.9 
atom-live-server 2.2.0 
atom-unsaved-changes 0.4.0 
autocomplete-paths 2.12.2 
block-comment 0.4.3 
build 0.70.0 
build-sass 0.11.0 
busy 0.7.0 
busy-signal 1.4.3 
color-picker 2.2.5 
console-log 1.10.0 
ctrl-dir-scroll 0.2.5 
cursor-blink-interval 0.3.1 
fake-image-snippets 1.1.0 
file-icons 2.1.17 
file-watcher 1.2.6 
filesize 4.0.0 
fonts 3.0.3 
goto-definition 1.3.4 
highlight-selected 0.13.1 
inc 0.0.4 
intentions 1.1.5 
jquery-snippets 11.0.0 
linter 2.2.0 
linter-eslint 8.4.1 
linter-htmlhint 1.4.0 
linter-jsonlint 1.3.0 
linter-markdown 5.2.0 
linter-stylelint 4.2.0 
linter-ui-default 1.6.10 
minimap 4.29.8 
minimap-bookmarks 0.4.2 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-linter 2.1.3 
minimap-pigments 0.2.2 
minimap-selection 4.5.0 
minimap-split-diff 0.3.7 
open-in-browser 0.5.2 
pigments 0.40.2 
project-manager 3.3.5 
random 1.4.1 
split-diff 1.5.2 
sync-settings 0.8.5 
toggle-quotes 1.1.0 
tree-view-autoresize 2.3.0 
```

**Comment by rsese** [comment]
Thanks for the report - this looks like a duplicate of #16630 so you can subscribe there for updates if you'd like.  I'll go ahead and copy over your comments over but do you happen to remember if you tabbed out of Atom  and then switched back when you saw the error?

---

Because we treat our issues list as the Atom team's backlog, we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see the How Can I Contribute? section of the Atom CONTRIBUTING guide.
